pkgs/http/lib/src/browser_client.dart

```diff
@@ -45,6 +45,10 @@ BaseClient createClient() {
 class BrowserClient extends BaseClient {
   final _abortController = AbortController();
 
+  BrowserClient([CacheOption? cacheOption]) {
```

Comment:
I think that this should probably be a keyword parameter, with a type of `String` and a default of `"default"`. You should also add documentation this this (check out `IOClient` for an example). You can link to https://developer.mozilla.org/en-US/docs/Web/API/Request/cache for the possible values.

You need to include tests in this PR.

Reply:
@brianquinlan Chcek the recent update.